.gitlab-ci-default-pipeline.yml

```diff
@@ -59,26 +59,39 @@ test:docs:
     DOCKER_CERT_PATH: "/certs/client"
     DOCKER_TLS_VERIFY: "1"
     DOCKER_TLS_CERTDIR: "/certs"
+    # Testing of different integration<->mender-docs branches matrix can be done
+    # by setting MENDER_DOCS_BRANCH (in format 'origin/branch_name')
+    # and INTEGRATION_BRANCH (in format 'branch_name') variables from GitLab UI
+    # when run the pipeline manually
```

Comment:
I thought this was code you added for debugging purposes only. What is the value in leaving it?

Reply:
It allows to run the pipeline from master and test any combination of `integration`-`mender-docs` branches. 
I'm not really sure about all the details of our release process, but assume it might be useful there. Another theoretical use case might be feature branches.

Reply:
Okey.

I think it is a bad practice to leave lingering code "just in case". It is better to remove that to keep code that we don't run in any automated way and hence might break without noticing.

In any case, if you want to keep it, please move it to the global variables section and set a more global name, like `TEST_DOCS_MENDER_DOCS_BRANCH`

Reply:
I would say it's up to you to decide here, because I have very little knowledge about releases and needs for testing it. If it doesn't make sense then definitely it worth to remove it.

Reply:
> I would say it's up to you to decide here, (...)

Remove it, then :smiley: